test/completion/test_tsserver_completion_parsing.vader

```diff
@@ -36,23 +36,23 @@ Execute(TypeScript completion details responses should be parsed correctly):
   \     'word': 'abc',
   \     'menu': '(property) Foo.abc: number',
   \     'info': '',
-  \     'kind': 'f',
+  \     'kind': 'v',
```

Comment:
These tests should still pass, as we should keep the existing defaults.

Reply:
Do we want to maintain strict backward compatibility? 

Seems to me we'd want to make these roughly match the kinds that vim has built in:

```
	v	variable
	f	function or method
	m	member of a struct or class
	t	typedef
	d	#define or macro
```

In particular, since this is a `property` it would probably map best to `m` (not `v` as it is right now, sorry). Same as the Rust types above. They probably map best to `t`, right?

I'm happy to maintain compat though or try this new logical mapping

Reply:
@w0rp sorry to bug you, but still waiting for a definitive response here :)